lldb/tools/lldb-dap/Handler/RequestHandler.h

```diff
@@ -47,7 +49,9 @@ class BaseRequestHandler {
 
   virtual void operator()(const protocol::Request &request) const = 0;
 
-  virtual llvm::StringMap<bool> GetCapabilities() const { return {}; }
+  virtual llvm::SmallDenseSet<AdapterFeature, 1> GetSupportedFeatures() const {
```

Comment:
Let's use a typedef to make this a little less verbose. 
```suggestion
  using FeatureSet =  llvm::SmallDenseSet<AdapterFeature, 1>;
  virtual FeatureSet GetSupportedFeatures() const {
```